docs/topics/sb-tomcat-runtime-overview.adoc

```diff
@@ -1,5 +1,5 @@
 = Runtime Details
-//https://projects.spring.io/spring-boot/ 
+//https://projects.spring.io/spring-boot/
 {SpringBoot} enables you to create opinionated Spring-based standalone applications. Check out xref:sb-tomcat-additional-resources[Additional Resources] for further reading on {SpringBoot}.
- 
-The {SpringBoot} runtime enables you to run {SpringBoot} applications and services in OpenShift while providing all the advantages and conveniences of the OpenShift platform such as rolling updates, service discovery, and canary deployments. OpenShift also makes it easier for your applications to implement common microservice patterns such as ConfigMap, Health Check, Circuit Breaker, and Failover.
\ No newline at end of file
+
+The {SpringBoot} runtime enables you to run {SpringBoot} applications and services in OpenShift while providing all the advantages and conveniences of the OpenShift platform such as rolling updates, service discovery, and canary deployments. OpenShift also makes it easier for your applications to implement common microservice patterns such as Externalized Configuration, Health Check, Circuit Breaker, and Failover.
```

Comment:
Here, too, externalized configuration seems like a common term, so I would advocate for using lower case (applies to all the patterns listed).